src/commands/showReferrer.ts

```diff
@@ -0,0 +1,86 @@
+import * as vscode from 'vscode';
+import { execAsync } from '../support/execAsync';
+import { window } from 'vscode';
+import * as os from 'os';
+import * as path from 'path';
+import { access } from 'fs/promises'; 
+
+const errorMessage = 'ORAS not found on default path "C:\\Users\\userName\\bin\\oras". Download ORAS or update path: ';
```

Comment:
This default path only applies to windows. Linux does not default paths with the drive letter